syntax = "proto3";

message Image {
    bytes data = 1;
}

message Id{
    string id = 1;
}

message Text{
    string text = 1;
}

service OCRServer{
    rpc ImageSync(Image) returns(Text){}
    rpc StoreImage(Image) returns(Id){}
    rpc GetText(Id) returns(Text){}
}

